backend/api/invoices/delete.py

```diff
@@ -39,4 +41,6 @@ def delete_invoice(request: HttpRequest):
         except Resolver404:
             return HttpResponseRedirect(reverse("dashboard"))
 
-    return JsonResponse({"message": "Invoice successfully deleted"}, status=200)
+    return JsonResponse({"message": "Invoice successfully deleted"}, status=200), QuotaLimit.delete_quota_usage(
```

Comment:
Why is the quota limit being returned? I feel like this isn't meant to be here

Reply:
Ahh, I was checking why `invoice.delete()` is marked as unreachable code and forgot to change it back.

Reply:
Yeah the unchangable code is due to pycharm, it's an active bug. I'm sure they've got a bug report that's still open from like 3 years ago or something

Reply:
I understand, code is back as it was now.